goreleaser.yaml

```diff
@@ -16,13 +16,14 @@ builds:
   - windows
   goarch:
   - amd64
-archive:
-  format: zip
-  files:
-  - manifest.json
-  - icon.png
-  - LICENSE
-  replacements:
+archives:
+  - id: output
+    format: zip
+    files:
+    - manifest.json
+    - icon.png
+    - LICENSE
+    replacements:
 
```

Comment:
![Screenshot from 2021-09-28 11-17-38](https://user-images.githubusercontent.com/30801614/135116104-f591a1bd-0ded-4e62-8423-8a5d5d5e22dd.png)
wants to use go 1.10 and does not recognizes subcommand `mod`